pkg/cfaws/profiles.go

```diff
@@ -35,6 +35,8 @@ type Profile struct {
 	AWSConfig    config.SharedConfig
 	Initialised  bool
 	LoadingError error
+	// set to true if aws temp credentails are fetched from ~/.aws/sso/cache dir.
+	IsDefaultCredential bool
```

Comment:
can we call this something like `UsePlaintextSSOToken`? Just thinking that "default" is a bit confusing, as Granted CLI defaults to using the keychain, whereas "default" in this context means "don't use the keychain".

```suggestion
	UsePlaintextSSOToken bool
```